src/eduid_userdb/db.py

```diff
@@ -239,7 +239,7 @@ def _get_all_docs(self):
         """
         return self._coll.find({})
 
-    def _get_document_by_attr(self, attr: str, value: str, raise_on_missing: bool = True) -> Optional[Mapping]:
+    def _get_document_by_attr(self, attr: str, value: str, raise_on_missing: bool = True) -> Optional[Dict]:
```

Comment:
I would think this change will be a problem for everything that expects a Mapping from _get_document_by_attr.